src/test/regress/expected/isolation_global_pid.out

```diff
@@ -30,17 +30,29 @@ run_commands_on_session_level_connection_to_node
 
 (1 row)
 
-step s2-coordinator-citus_dist_stat_activity:
-    SELECT global_pid != 0 FROM citus_dist_stat_activity() WHERE query LIKE '%SELECT * FROM dist\_table%' and query NOT ILIKE '%run_commands_on_session_level_connection_to_node%';
+step s2-coordinator-citus_stat_activity:
+    SELECT global_pid != 0 FROM citus_stat_activity WHERE query LIKE '%SELECT * FROM dist\_table%' AND query NOT ILIKE '%run_commands_on_session_level_connection_to_node%';
```

Comment:
maybe this PR is a good one to remove all citus_dist/worker_stat_activity  from the tests? Or, do you want to do it in another PR?

Reply:
There are some more tests so I'd prefer a separate PR.